### ISSUE TITLE
Could you tell me the approximate final train_loss?

### ISSUE THREAD

**Issue by zldrobit** [zackthoutt/got-book-6#18]
In the notebook, the last train_loss is approximately 2.4. 
Since I am training a language generating model like yours, i am very curious about the final train_loss you get.
Thanks!

**Comment by zackthoutt** [comment]
I'm not sure what my exact final training loss was. I trained the model in July and had several GPU instances running with different hyperparameters and chose one of the better performing ones to post in the repo. It was probably around 2.4 though.
